Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -129,53 +127,6 @@ public async Task<FindCloudletReply> FindCloudlet()
             return reply;
         }
 
-        /// <summary>
```

Comment:
I sort of still want it, perhaps as an example to be customized to mean something to a particular developer. Likely do so once we implement events (Location too far, etc).

Reply:
Added back